doc/modules/model_evaluation.rst

```diff
@@ -102,11 +102,27 @@ Scoring                                Function
 'neg_mean_poisson_deviance'            :func:`metrics.mean_poisson_deviance`
 'neg_mean_gamma_deviance'              :func:`metrics.mean_gamma_deviance`
 'neg_mean_absolute_percentage_error'   :func:`metrics.mean_absolute_percentage_error`
-'d2_absolute_error_score'              :func:`metrics.d2_absolute_error_score`
-'d2_pinball_score'                     :func:`metrics.d2_pinball_score`
-'d2_tweedie_score'                     :func:`metrics.d2_tweedie_score`
 ====================================   ==============================================     ==================================
 
+The following metrics functions are not implemented as named
+scorers. They cannot be passed to the ``scoring`` parameters; instead
+their callable needs to be passed to :func:`make_scorer` together with the
+value of the user-settable parameters. See :ref:`scoring` for more
+details on :func:`make_scorer` usage.
+
+=====================================  =========  ==============================================
+Function                               Parameter  Example usage
+=====================================  =========  ==============================================
+**Classification**
+:func:`metrics.fbeta_score`            ``beta``   ``make_scorer(fbeta_score, beta=2)``
+
+**Regression**
+:func:`metrics.mean_tweedie_deviance`  ``power``  ``make_scorer(mean_tweedie_deviance, power=1)``
```

Comment:
I would use 1.5 or 1.8 for the power parameter. `power=1` is equivalent to the Poisson deviance in which case it would be more natural to use the named `"neg_mean_poisson_deviance"`. `power=2` would be the gamma deviance.